docs/known-issues.md

```diff
@@ -22,4 +22,6 @@ Known issue = any major user-facing bug or lack of user-facing feature that:
 | Deleting entities directly from the Inspector causes undefined behavior. | 2018-07-17 | [UNR-425](https://improbableio.atlassian.net/projects/UNR/issues/UNR-425) | | No |
 | Interop code generation stalls if its child processes produce console output during execution. |2018-07-19 | [UNR-422](https://improbableio.atlassian.net/browse/UNR-422) |  | No |
 | Having generated files in the symlinked folder `Source/SpatialGDK` means you can’t work in two projects at the same time using the same clone of the SpatialOS Unreal GDK. | 2018-07-23 | [UNR-458](https://improbableio.atlassian.net/browse/UNR-458) | Clone the Unreal GDK separately for each project. | No |
-| We don't support listen servers. | 2018-07-30 | | Use dedicated servers instead. | No |
\ No newline at end of file
+| We don't support listen servers. | 2018-07-30 | | Use dedicated servers instead. | No |
+| Attempting to use stably-named replicated objects (ie. placed level actors marked for replication) will result in runtime assertions. | 2018-07-31 | [UNR-397](https://improbableio.atlassian.net/browse/UNR-397) | For now, please spawn Actor at runtime instead. | No
+| Attempting to find the bound class of a blueprint class' typebinding fails if the blueprint hasn't previously been loaded. | 2018-07-31 | [UNR-273](https://improbableio.atlassian.net/browse/UNR-273) | Add a reference to the blueprint in default level. | No
```

Comment:
I think a better workaround is "open the blueprint in editor"

Reply:
That will work for the interop generator, but at runtime you could still be missing your class. Adding to level is clumsy, but catches both instances.

Reply:
Why would you miss it in runtime if your game uses the class in some way? For example a blueprint based player controller.

Reply:
The class has to be loaded when we run `USpatialInterop::Init`. Perhaps it's delayed? The bug was initially reported as happening at runtime.

Reply:
Hmm ok, I miscategorized the bug I'm seeing then. UNR-273 should be split in two.

Reply:
Yes it can occur when generating typebindings, or at interop on startup. Both stem from the same issue though, so they are linked somewhat, and the fix for both would probably go in at the same time.